[CI] GenerativeIT test failing

**Build Scans:**
- [elasticsearch-intake #34219 / part3](https://gradle-enterprise.elastic.co/s/dqnkdklapxr7i)
- [elasticsearch-pull-request #118784 / part-3](https://gradle-enterprise.elastic.co/s/pmfabhwxt7jru)
- [elasticsearch-pull-request #118770 / part-3](https://gradle-enterprise.elastic.co/s/4epivwqhufvjg)
- [elasticsearch-pull-request #118754 / part-3](https://gradle-enterprise.elastic.co/s/wvdmhynydovuq)
- [elasticsearch-pull-request #118299 / part-3](https://gradle-enterprise.elastic.co/s/akk4y2rqqfngs)
- [elasticsearch-pull-request #118377 / part-3](https://gradle-enterprise.elastic.co/s/bizgdn3zodkhg)
- [elasticsearch-periodic-platform-support #11809 / ubuntu-2204_platform-support-unix](https://gradle-enterprise.elastic.co/s/cz6lpp3mcqvva)
- [elasticsearch-periodic-platform-support #11809 / ubuntu-2404_platform-support-unix](https://gradle-enterprise.elastic.co/s/nltgma6giylpc)
- [elasticsearch-pull-request #117835 / part-3](https://gradle-enterprise.elastic.co/s/l7i7tmsotsgzs)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:multi-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.multi_node.GenerativeIT.test" -Dtests.seed=B064158A2C16BF1 -Dtests.locale=vi -Dtests.timezone=Asia/Dhaka -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.multi_node.GenerativeIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: query: from apps_short,countrie*,many_numbers | sort name NULLS FIRST, version, value | drop `shape` | enrich languages_policy on name | enrich languages_policy on language_name | enrich languages_policy on language_name | grok name "%{WORD:version} %{WORD:zsmISHrIx}" | inline stats  zsmISHrIx = present(name), language_name = values(value) by zsmISHrIx | stats  errXodFfffWu = absent(value + language_name), name = median(language_name), id = median(language_name) + max(language_name) + median(language_name), VXkRCYil = count(value)
exception: 60.000 milliseconds timeout on connection http-outgoing-1 [ACTIVE]
```

**Issue Reasons:**
- [main] 9 failures in test test (1.4% fail rate in 628 executions)
- [main] 6 failures in step part-3 (1.5% fail rate in 393 executions)
- [main] 6 failures in pipeline elasticsearch-pull-request (1.5% fail rate in 392 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -270,9 +270,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT
   method: test {csv-spec:spatial.ConvertFromStringParseError}
   issue: https://github.com/elastic/elasticsearch/issues/139213
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT
-  method: test {csv-spec:string.Url_encode_component tests with table reads}
-  issue: https://github.com/elastic/elasticsearch/issues/140809
 - class: org.elasticsearch.xpack.esql.heap_attack.HeapAttackSubqueryIT
   method: testGiantTextFieldInSubqueryIntermediateResultsWithSort
   issue: https://github.com/elastic/elasticsearch/issues/141034
@@ -368,9 +365,6 @@ tests:
 - class: org.elasticsearch.xpack.test.rest.XPackRestIT
   method: test {p0=esql/50_index_patterns/disjoint_mappings}
   issue: https://github.com/elastic/elasticsearch/issues/141856
-- class: org.elasticsearch.xpack.esql.qa.multi_node.GenerativeIT
-  method: test
-  issue: https://github.com/elastic/elasticsearch/issues/141894
 - class: org.elasticsearch.compute.lucene.read.ValueSourceReaderTypeConversionTests
   method: testLoadAll
   issue: https://github.com/elastic/elasticsearch/issues/141954

